shake.yaml

```diff
@@ -0,0 +1,7 @@
+# Used to provide a different environment for the shake build script
+resolver: nightly-2018-12-01 # GHC 8.6.2
```

Comment:
Is there any way we can do something like "use the system resolver on this machine"?

Reply:
An extra `stack.yaml` is required since the other `yaml`s contain information about extra dependencies in the `submodules` folder, which is not present on a non-recursive clone or if the submodules change. Since the build script should take care of updating the git submodules, they are not allowed to be referenced within the `stack-yaml` that is used by `Shakefile.hs`.

It is possible, however, to include `system-ghc: true` within the `shake.yaml` ([see option](https://docs.haskellstack.org/en/stable/yaml_configuration/#system-ghc)) However, this seems to be incompatible with Docker and nix.